libafl_libfuzzer/libafl_libfuzzer_runtime/src/fuzz.rs

```diff
@@ -50,7 +50,7 @@ pub fn fuzz(
 
         match Launcher::builder()
             .shmem_provider(shmem_provider)
-            .configuration(EventConfig::from_name("default"))
+            .configuration(EventConfig::AlwaysUnique)
```

Comment:
Why not create that according to the options?

Reply:
Oops, that one was testing :sweat_smile: It should always be the same for libfuzzer targets since we're restricted to a particular set of observers.